streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/SubscriptionInfoTest.java

```diff
@@ -41,19 +41,6 @@
 
     private final static String IGNORED_USER_ENDPOINT = "ignoredUserEndpoint:80";
 
-    @Test
-    public void shouldUseLatestSupportedVersionByDefault() {
```

Comment:
Didn't realise this before, but this test was verifying a test-only constructor that has already been inlined, so the test is now pointless.